artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/QueueControlImpl.java

```diff
@@ -1722,6 +1722,21 @@ public boolean isPaused() throws Exception {
       }
    }
 
+   @Override
+   public boolean isPersistedPause() {
+      if (AuditLogger.isBaseLoggingEnabled()) {
+         AuditLogger.isPaused(queue);
```

Comment:
It seems like this would need to have a new audit logger call added and not call the isPasued API

Reply:
Fixed.